docs/shared/getting-started/intro.md

```diff
@@ -4,53 +4,25 @@ Nx is a smart, fast and extensible build system with first class monorepo suppor
 
 ## Philosophy
 
-Nx has a similar design philosophy to Visual Studio Code. Code is a powerful text editor, and you can be very productive
-with it even if you don't install any extensions. The ecosystem of VSCode's extensions though is what can really level
-up your productivity.
+Nx shares the design philosophy with Visual Studio Code. Code is a powerful text editor, and you can be very productive
+with it even if you don't install any extensions. The ecosystem of VSCode's extensions is what levels up your productivity.
 
-Nx is similar. The core of Nx is generic, simple, and unobtrusive. Nx plugins, although very useful for many projects,
-are completely optional.
+Similarly, the core of Nx is generic, simple, and unobtrusive. Nx plugins, although very useful, are completely optional.
+(Placeholder)
+Because Nx generates the boilerplate for you, most examples use Nx plugins making it easier to demonstrate the features of Nx. The vast majority of features work without the need of plugins.
 
-Most examples on this site use Nx plugins. It's just easier to demonstrate many features Nx offers when Nx generates all
-the boilerplate. However, the vast majority of the features will work the same way in a workspace with no plugins.
+## (PlaceHolder) Why should someone care about nx? What does nx bring to the table?
 
-## Getting Started
+> is this where we talk about this?
 
-These guides will help you get started:
+- think (ESBuild)[https://esbuild.github.io/] time to build graph. straight to the point of "this is what we do and
+- solve"
+- why do people care about monorepo tooling?
+- my code is not in a monorepo, so why should I care?
+- or can I even use this outside of a monorepo
 
-- [Installing Nx CLI & creating a new Nx Workspace](/getting-started/nx-setup)
-- [Adding Nx to an existing monorepo](/migration/adding-to-monorepo)
-- [Using Nx without plugins](/getting-started/nx-core)
-- [Nx and TypeScript](/getting-started/nx-and-typescript)
-- [Nx and React](/getting-started/nx-and-react)
-- [Nx and Angular](/getting-started/nx-and-angular)
-- [CI Overview](/using-nx/ci-overview)
```

Comment:
The first thing people will ask when getting to the docs is
"how do I use it?"
or
 "I'm interested but want to know more first"
I think this is a good point to give a really nice hook into how powerful nx can be. showing off the features vs just listing them would be better. a quick 60 second video or visual showcase of the nx ecosystem and how it can "level up" people would be good or something that extends what was talked about on the home page.